validation-test/stdlib/Set.swift

```diff
@@ -138,7 +138,7 @@ func isNativeSet<T : Hashable>(_ s: Set<T>) -> Bool {
 #if _runtime(_ObjC)
 func isNativeNSSet(_ s: NSSet) -> Bool {
   let className: NSString = NSStringFromClass(type(of: s)) as NSString
-  return className.range(of: "NativeSetStorage").length > 0
+  return className.range(of: "NativeSetBridgingStorage").length > 0
```

Comment:
I realize this isn't part of the change, but comparing class name strings *really* seems like the wrong way to do this test.  If it isn't wrong, there should be a comment explaining why not.